libraries/Microsoft.Bot.Builder.LanguageGeneration/TemplateEngine.cs

```diff
@@ -75,6 +75,12 @@ public TemplateEngine AddFiles(IEnumerable<string> filePaths, ImportResolverDele
         /// <returns>Template engine with the parsed content.</returns>
         public TemplateEngine AddText(string content, string name, ImportResolverDelegate importResolver)
         {
+            // fix https://github.com/microsoft/botbuilder-dotnet/issues/2249
```

Comment:
Require a id and importResolver because
copy the text of issue here. // if no importResolver provided, fallback to fileResolver which is largely not compatible with the id here.